packages/core/admin/ee/server/controllers/workflows/index.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const { merge } = require('lodash/fp');
+const { getService, mapObject } = require('../../utils');
+
+function sanitizeWorkflowQuery(query = {}) {
+  return mapObject(query, {
+    pick: ['workflow_id'],
+    rename: { workflow_id: 'id' },
+  });
+}
+
+module.exports = {
+  /**
+   * List all workflows
+   * @param {import('koa').BaseContext} ctx - koa context
+   */
+  async find(ctx) {
+    const query = sanitizeWorkflowQuery(merge(ctx.query, ctx.params));
+
+    const workflowService = getService('workflows');
+    const results = await workflowService.find(query);
+
+    ctx.body = {
+      results,
```

Comment:
We're trying to keep consistency across the different endpoints by returning a `data` property as the successful result of an HTTP request 

```suggestion
      data: results,
```

Reply:
Actually, it was to remain coherent with admin requests on content-manager.

An example of data in the content-manager:
```json
{
  "results": [
    {
      "id": 1,
      "postal_code": "06",
      "city": "Le Cannet",
      "json": null,
      "slug": null,
      "createdAt": "2022-12-07T10:46:30.361Z",
      "updatedAt": "2022-12-07T10:54:43.019Z",
      "categories": {
        "count": 2
      },
      "cover": null,
      "images": null,
      "notrepeat_req": {
        "id": 1,
        "name": "toto"
      },
      "repeat_req": [
        {
          "id": 2,
          "name": "toto"
        }
      ],
      "repeat_req_min": [],
      "createdBy": {
        "id": 1,
        "firstname": "Nathan",
        "lastname": "Pichon",
        "username": null
      },
      "updatedBy": {
        "id": 1,
        "firstname": "Nathan",
        "lastname": "Pichon",
        "username": null
      }
    },
    {
      "id": 2,
      "postal_code": "42",
      "city": "Saint Etienne",
      "json": null,
      "slug": null,
      "createdAt": "2022-12-07T10:47:13.465Z",
      "updatedAt": "2022-12-07T10:54:56.061Z",
      "categories": {
        "count": 2
      },
      "cover": null,
      "images": null,
      "notrepeat_req": {
        "id": 3,
        "name": "toto"
      },
      "repeat_req": [
        {
          "id": 4,
          "name": "toto"
        }
      ],
      "repeat_req_min": [],
      "createdBy": {
        "id": 1,
        "firstname": "Nathan",
        "lastname": "Pichon",
        "username": null
      },
      "updatedBy": {
        "id": 1,
        "firstname": "Nathan",
        "lastname": "Pichon",
        "username": null
      }
    }
  ],
  "pagination": {
    "page": 1,
    "pageSize": 10,
    "pageCount": 1,
    "total": 2
  }
}
```

Reply:
Yeah lists don't return a data attribute. That being said: for the FE this creates a problem already. To not have it seems more consistent in the BE, but it would be nice if we could start to have the response in `data` everywhere, but I guess that is a wider discussion.

Reply:
I didn't know the content manager is using results instead of data, that's interesting, thanks for pointing that out :thinking: 
However, is your route a content manager one or does it belong to regular admin routes?

Reply:
It's not a content manager route.
I wouldn't mind to rename results in data. Should we plan a task to refactor content-manager routes as well?

Reply:
I'd like that, but for me it can be a follow-up task which can be done after this iteration.

Reply:
It depends if there is pagination. When there is pagination it's `results, pagination`, otherwise it's `data`. A bit weird but that the logic for the moment

Reply:
https://strapi-inc.atlassian.net/browse/CONTENT-999

Reply:
No pagination: all stages should be returned :)